src/clients/private/orgs/subscriptions/OrgSubscriptionDetailPage/SubscriptionPaymentTab/BottomActionSection/SplitBillingHistoryModal.tsx

```
@@ -0,0 +1,106 @@
+                                </section>
+                            </div>
+                        </div>
+                        <PayAmount billingHistory={latest} className="font-semibold text-gray-900 whitespace-nowrap" />
+                    </div>
+                </section>
+                <section className="flex gap-2">
+                    <button className="btn btn-white btn-md btn-block flex-1" onClick={onClose}>
+                        취소
+                    </button>
+                    <button
+                        className={`btn btn-scordi btn-md btn-block flex-1 ${isLoading ? 'link_to-loading' : ''}`}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)